### ISSUE TITLE
Protein2GO reports

### ISSUE THREAD

**Issue by RLovering** [ebi-uniprot/Protein2GO#19]
Hi Tony

please could this be added to the wish list:

There are loads of reports and it is difficult to find the one we want, and we don’t always know if the report includes D/I/U data or just actual released data. 

Rachael wondered if the SQL query could be shown (somehow) so that we can understand what the data source is better. 

I wondered if the list could be rearranged and put under some sort of subheadings to list the different queries e.g. PubMed IDs/reference (maybe standardise this to either PubMed ID or Reference) ; annotations ; activity ; entities. Although maybe that is too difficult or too time consuming for little reward?

Thanks

Ruth

**Comment by tonysawfordebi** [comment]
I agree that the number of reports is getting a bit unwieldy, and maybe a bit of reorganisation is called for.

And trust me when I say that you *really* don't want to see the SQL that drives these reports - it's the stuff of nightmares. No, actually it's worse than that.
